sentry-openfeign/src/main/java/io/sentry/openfeign/SentryFeignClient.java

```diff
@@ -50,8 +51,13 @@ public Response execute(final @NotNull Request request, final @NotNull Request.O
       span.setDescription(request.httpMethod().name() + " " + request.url());
 
       final SentryTraceHeader sentryTraceHeader = span.toSentryTrace();
+      final @Nullable BaggageHeader baggageHeader = span.toBaggageHeader();
       final RequestWrapper requestWrapper = new RequestWrapper(request);
       requestWrapper.header(sentryTraceHeader.getName(), sentryTraceHeader.getValue());
```

Comment:
Should we check for `TracingOrigins.contain(hub.options.tracingOrigins, request.url.toString())` here as well? We do in other places.

Reply:
Yeah, I guess it was just missed. What @maciejwalkowiak would say?